.github/workflows/ci.yml

```
@@ -83,4 +83,4 @@ jobs:
         run: ./tests/bin/install.sh woo_test root root 127.0.0.1 ${{ matrix.wp }}
 
       - name: Run tests
-        run: pnpm -- turbo run test --filter=woocommerce
+        run: pnpm -- turbo run turbo:test --color --filter=woocommerce
```

```suggestion
        run: pnpm run test --filter="woocommerce"
```

Like the above, it feels more maintainable to rely on PNPM's semantics since we're abstracting Turbo. This one shouldn't have the same output weirdness because `"turbo:test"` doesn't run on dependencies.